python_modules/libraries/dagster-dbt/dagster_dbt/cli/resources_v2.py

```diff
@@ -232,6 +233,30 @@ def get_asset_key_for_model(
 
         return self.node_info_to_asset_key(next(iter(matching_models)))
 
+    def create_asset_selection_for_dbt_selection(
```

Comment:
We tend to use "build" instead of create.  E.g. `DynamicPartitionsDefinition.build_add_request` and `build_op_context` etc.

Also, I wonder if there's some way we can make this name less verbose without sacrificing clarity. Would there be easy ways to misinterpret it if it was just `build_asset_selection`? Would it help if we named the parameter `dbt_select`?